src/util/FileHelper.ts

```diff
@@ -79,6 +78,21 @@ export default class FileHelper {
     return `${fileSizeInBytes} bytes`
   }
 
+  public static costString(bytes: number, weiPerByte: number): string {
+    const costWei = weiPerByte * bytes
+    const costETH = costWei / 1000000000000000000
```

Comment:
We can use `web3`'s method to convert from wei_value to eth here: 

`web3.utils.fromWei(number [, unit])`

[doc](https://web3js.readthedocs.io/en/v1.2.0/web3-utils.html#fromwei)

Reply:
ok cool